rawkit/raw.py

```diff
@@ -175,6 +175,58 @@ def save_thumb(self, filename=None):
         self.libraw.libraw_dcraw_thumb_writer(
             self.data, filename.encode('ascii'))
 
+    def get_4_col_raw(self):
```

Comment:
Hi.

It is easy to change this to the other case (with only 3 colours), as the data then is in `color3_image` and not in `color4_image`. Though before I modify this I would like to know what the return types should be. In my mind we should return something that is 2D, as this removes the need to pass along height and width.